packages/components/src/components/list/Sortable/SortableItem.tsx

```diff
@@ -73,7 +73,8 @@ function collectTarget(connect: reactDnd.DropTargetConnector, monitor: reactDnd.
   };
 }
 
-class Item extends React.PureComponent<WrappedSortableItemProps, {}> {
+class Item extends React.PureComponent<WrappedSortableItemProps, Record<string, unknown>> {
+  // eslint-disable-next-line react/state-in-constructor
```

Comment:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/state-in-constructor.md